contracts/modules/LoanMaintenance/LoanMaintenance.sol

```diff
@@ -25,8 +25,8 @@ contract LoanMaintenance is State, LoanMaintenanceEvents, VaultController, Inter
         _setTarget(this.depositCollateral.selector, target);
         _setTarget(this.withdrawCollateral.selector, target);
         _setTarget(this.setDepositAmount.selector, target);
-        _setTarget(this.getUserLoans.selector, target);
```

Comment:
this will break our chainlink keepers used for liquidations.

Reply:
fixed